### DISCUSSION TITLE
Placeholder font size

### DISCUSSION THREAD

**Discussion by christiansavels** [slint-ui/slint#2827] [Ideas] [answered]
Hello,

I couldn't find a parameter to set the placeholder size.
Why not give it the same size as the input text? Just a thought ;)

`export component App inherits Window {
    VerticalBox {
        LineEdit {
            font-size: 48px;
            placeholder-text: "my placeholder";
        }
        Button {
            text: "Submit";
        }
    }
 }`
![image](https://github.com/slint-ui/slint/assets/71607651/279bc9c3-3fed-437a-8218-03eeb533bd7b)
![image](https://github.com/slint-ui/slint/assets/71607651/94970bf2-f8c5-45cd-8eaf-f92d530fc7e7)

**Comment by Xiaobaishushu25** [comment]
i also has this question，and i think LineEdit should allow us to set the text color, border color, and placeholder text color.Is it possible to achieve this？

**Reply by tronical** [reply]
Ahh, you're right - well spotted. The placeholder should share the font-size and other attributes.

**Reply by tronical** [reply, marked as answer]
Tracking in https://github.com/slint-ui/slint/issues/2829
